src/lib/components/Preview.svelte

```
@@ -47,24 +28,30 @@
+	$: canChangeStatus =
+		isAdmin ||
+		(isOwner && (selectedRfd?.status === 'draft' || selectedRfd?.status === 'open_for_review'));
 	$: isDraft = selectedRfd?.status === 'draft';
 	$: isOpenForReview = selectedRfd?.status === 'open_for_review';
-	
+
 	// Status description reactive variables
 	$: currentStatusDescription = selectedRfd ? getStatusDescription(selectedRfd.status) : null;
-	$: showStatusDescription = selectedRfd ? shouldShowStatusDescription(selectedRfd.status, isOwner, isAdmin) : false;
-	$: statusColorScheme = currentStatusDescription ? getStatusColorScheme(currentStatusDescription.colorScheme) : null;
+	$: showStatusDescription = selectedRfd
```

ESLint

[error] 'showStatusDescription' is defined but never used.

(@typescript-eslint/no-unused-vars)